site/content/includes/installation/install-gateway-api-experimental-features.md

```diff
@@ -0,0 +1,23 @@
+---
+docs: "DOCS-000"
+---
+
+To use Gateway API experimental resources, the Gateway API resources from the experimental channel must be installed before deploying NGINX Gateway Fabric. Additionally, NGINX Gateway Fabric must have experimental features enabled.
+
+{{<caution>}}As noted in the [Gateway API documentation](https://gateway-api.sigs.k8s.io/guides/#install-experimental-channel), future releases of the Gateway API can include breaking changes to experimental resources and fields.{{</caution>}}
+
+To install the Gateway API resources from the experimental channel, run the following:
+
+```shell
+kubectl kustomize "https://github.com/nginxinc/nginx-gateway-fabric/config/crd/gateway-api/experimental?ref=v1.4.0" | kubectl apply -f -
+```
+
+{{<note>}}If you plan to use the `edge` version of NGINX Gateway Fabric, you can replace the version in `ref` with `main`, for example `ref=main`.{{</note>}}
+
+To enable experimental features on NGINX Gateway Fabric:
+
+If using Helm: The `nginxGateway.gwAPIExperimentalFeatures.enable` option must be set to true. An example can be found
+in the [Installation with Helm]({{< relref "installation/installing-ngf/helm.md#custom-installation-options" >}}) guide.
+
+If using Kubernetes manifests: The `--gateway-api-experimental-features` command-line flag must be set to true on the deployment manifest.
+An example can be found in the [Installation with Kubernetes manifests]({{< relref "installation/installing-ngf/manifests.md#3-deploy-nginx-gateway-fabric" >}}) guide.
```

Comment:
```suggestion
Using Helm: Set `nginxGateway.gwAPIExperimentalFeatures.enable` to true. An example can be found
in the [Installation with Helm]({{< relref "installation/installing-ngf/helm.md#custom-installation-options" >}}) guide.

Using Manifests: Add the  `--gateway-api-experimental-features` command-line flag to the deployment manifest.
An example can be found in the [Installation with Kubernetes manifests]({{< relref "installation/installing-ngf/manifests.md#3-deploy-nginx-gateway-fabric" >}}) guide.
```

Changes for conciseness. The Kubernetes instruction might be wrong: I assume that the argument is implicitly true, and doesn't have additional parameters.

Reply:
Made the change though its slightly adjusted, your assumption is correct.

Reply:
Fair enough! In context there's no other kind of Manifest it could be mistaken for, so I favour the shorter noun.

The specificity is more important for components with more generic or shared names.